### DISCUSSION TITLE
Where is bimmer_connected.json when CLI is called on Mac?

### DISCUSSION THREAD

**Discussion by ryanbuckner** [bimmerconnected/bimmer_connected#685] [General] [answered]
The documentation states that "The CLI will store your tokens in a file called `bimmer_connected.json` in your home directory (--oauth-store parameter). You can move this file, but is is required to function properly (have multiple calls). The data is stored in plain text, so protect this file accordingly."

When I run a CLI command using terminal on my Mac, where is this file stored?  Running a search does not find it.

**Comment by rikroe** [comment, marked as answer]
Using Finder, it'll be the directory of your user (i.e. where Desktop/Documents/etc. are located). 
Please note that the `.` at the beginning indicates that the files is a hidden file, so you need to check how to open it.

Otherwise you can just use the MacOS terminal and type `cat ~/.bimmer_connected.json`.

I have seen that the docs are not looking as I would have wanted them and will rework them.

**Comment by ryanbuckner** [comment]
Thanks. I since it's hidden that's been my problem. I found it.
